cmd/list/machinepool/machinepool.go

```diff
@@ -68,6 +93,7 @@ func listMachinePools(r *rosa.Runtime, clusterKey string, cluster *cmv1.Cluster)
 			printStringSlice(machinePool.AvailabilityZones()),
 			printStringSlice(machinePool.Subnets()),
 			printSpot(machinePool),
+			printMachinePoolDiskSize(machinePool, defaultRootDiskSize),
```

Comment:
@ciaranRoche @renan-campos do we want this code?

Reply:
It's going to print the default flavor value if nothing is set:

```
$ rosa list machinepools -c 258idq729qn7g8ltl01vjsiopopf8lmi
ID            AUTOSCALING  REPLICAS  INSTANCE TYPE  LABELS    TAINTS    AVAILABILITY ZONES    SUBNETS    SPOT INSTANCES  DISK SIZE
Default       No           2         m5.xlarge                          us-west-2a                       N/A             256 GiB
mp-ocm-15     No           2         m5.xlarge                          us-west-2a                       No              128 GiB
no-disk-size  No           2         m5.xlarge                          us-west-2a                       No              300 GiB
```

I personally think it makes, especially if you want multiple MP.
Thoughts?

Reply:
Resolving, as per the recent discussion, we are setting the "default" keyword in that column.